components/admin_console/schema_text.jsx

```diff
@@ -38,6 +38,8 @@ export default class SchemaText extends React.PureComponent {
 
         if (isMarkdown) {
             return (
+
+                // Give all markdown links target="_blank"
```

Comment:
This comment seems a little unrelated to the purpose of this file/block of code, and adds a little bit of noise to readability. Thoughts on removing this comment @Borknab?

Reply:
Agreed. Doesn't make sense, thanks for pointing that out!